### PR TITLE
Update SDC host to use remote location

### PR DESCRIPTION
**Pull request by shekhar-j**
# Description
Update SDC host to use remote location

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [X] I have maintained at least 80% code coverage
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by shekhar-j** [comment]
run e2e tests

**Comment by shekhar-j** [comment]
run e2e test

**Comment by shekhar-j** [comment]
run e2e tests

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@ HOSTNAME=registry.terraform.io
 NAMESPACE=dell
 NAME=powerflex
 BINARY=terraform-provider-${NAME}
-VERSION=1.5.0
+VERSION=1.5.1
```
Comment (Krunal-Thakkar) [inline Makefile:6]: it will be 1.6.0, correct?

**Comment by rounak-adhikary** [comment]
`use_remote_path` applies to windows too, rt? Then we need to make changes in Windows as well.

**Review by Krunal-Thakkar — APPROVED** [review]

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

**Review by rounak-adhikary — APPROVED** [review]
LGTM

**Review by doriac11 — APPROVED** [review]

**Pull request merged**